src/test/java/org/fundacionjala/coding/daniel/Kata8Test.java

```diff
@@ -0,0 +1,104 @@
+package org.fundacionjala.coding.daniel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Class for Testing Kata8 class.
+ */
+public class Kata8Test {
+    private Kata8 kata;
+
+    /**
+     * This method initializes variable kata.
+     */
+    @Before
+    public void init() {
+        kata = new Kata8();
+    }
+
+    /**
+     * This method tests mapping of an account number given
+     * in pipes and underscores to string of digits.
+     */
+    @Test
+    public void testMapEntryOCRToNumber() {
+        List<String> lines = new ArrayList<>();
+        lines.add("    _  _     _  _  _  _  _ ");
+        lines.add("  | _| _||_||_ |_   ||_||_|");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "  | _| _||_||_ |_   ||_||_|" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)